mustang_codegen/lib/src/utils.dart

```diff
@@ -90,6 +91,33 @@ class Utils {
     return null;
   }
 
+  /// Input: MethodElement
+  /// Output: Method with all its input arguments
+  /// Example Output: validateToken(userId: userId, token: token)
+  static String methodWithExecutionArgs(
+    MethodElement element,
+    List<String> imports,
+  ) {
+    String methodName = 'super.${element.displayName}(';
```

Comment:
this should be named methodBody ?

Reply:
`methodBody` isn't accurate because it's not actually the method's body. I think `methodWithArguments` is more accurate